src/codeu/chat/common/NetworkCode.java

```diff
@@ -35,5 +35,7 @@
       RELAY_READ_REQUEST = 27,
       RELAY_READ_RESPONSE = 28,
       RELAY_WRITE_REQUEST = 29,
-      RELAY_WRITE_RESPONSE = 30;
+      RELAY_WRITE_RESPONSE = 30,
+      SERVER_INFO_REQUEST = 31,
+          SERVER_INFO_RESPONSE = 32;
```

Comment:
Remove this indent so that it's on the same level with the rest of the constants - it's good to keep formatting as consistent as possible